app/javascript/packages/normalize-yaml/cli.js

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env node
+import { promises as fsPromises } from 'fs';
+import { join } from 'path';
+import prettier from 'prettier';
+import normalize from './index.js';
+
+const { readFile, writeFile } = fsPromises;
+
+/** @type {Record<string,any>=} */
+const prettierConfig = prettier.resolveConfig.sync(process.cwd());
+
+const files = process.argv.slice(2);
```

Comment:
Toyed with supporting piped input to avoid the multi-step `make update_country_dialing_codes`. Works, but keeping as-is since it's a bit simpler.

```diff
diff --git a/Makefile b/Makefile
index b9edbdcc9..61c96ae61 100644
--- a/Makefile
+++ b/Makefile
@@ -87,8 +87,7 @@ lint_optimized_assets: optimize_assets
 	(! git diff --name-only | grep "\.svg$$") || (echo "Error: Optimize assets using 'make optimize_assets'"; exit 1)
 
 update_country_dialing_codes:
-	bundle exec ./scripts/pinpoint-supported-countries > config/country_dialing_codes.yml
-	yarn normalize-yaml config/country_dialing_codes.yml
+	bundle exec ./scripts/pinpoint-supported-countries | yarn run --silent normalize-yaml > config/country_dialing_codes.yml
 
 check_asset_strings:
 	find ./app/javascript -name "*.js*" | xargs ./scripts/check-assets
diff --git a/app/javascript/packages/normalize-yaml/cli.js b/app/javascript/packages/normalize-yaml/cli.js
index 9c67362d4..8b0afbb2a 100755
--- a/app/javascript/packages/normalize-yaml/cli.js
+++ b/app/javascript/packages/normalize-yaml/cli.js
@@ -4,16 +4,26 @@ import { join } from 'path';
 import prettier from 'prettier';
 import normalize from './index.js';
 
-const { readFile, writeFile } = fsPromises;
-
 /** @type {Record<string,any>=} */
 const prettierConfig = prettier.resolveConfig.sync(process.cwd());
 
-const files = process.argv.slice(2);
-Promise.all(
-  files.map(async (relativePath) => {
-    const absolutePath = join(process.cwd(), relativePath);
-    const content = await readFile(absolutePath, 'utf8');
-    await writeFile(absolutePath, normalize(content, prettierConfig));
-  }),
-);
+if (process.stdin.isTTY) {
+  const { readFile, writeFile } = fsPromises;
+  const files = process.argv.slice(2);
+  Promise.all(
+    files.map(async (relativePath) => {
+      const absolutePath = join(process.cwd(), relativePath);
+      const content = await readFile(absolutePath, 'utf8');
+      await writeFile(absolutePath, normalize(content, prettierConfig));
+    }),
+  );
+} else {
+  (async () => {
+    let content = '';
+    for await (const chunk of process.stdin) {
+      content += chunk.toString();
+    }
+
+    process.stdout.write(normalize(content, prettierConfig));
+  })();
+}
```